Wright_Will/Assignments/underScore/underScore.py

```diff
@@ -0,0 +1,47 @@
+class underScore(object):
+    def __init__(self):
+        super(underScore, self).__init__()
+        self.pi = 3.14159
+    #maps an iterble in place
+    def map(self,iterbl,fn):
+        for i in range(len(iterbl)):
```

Comment:
looks good. i'd still use a list comprehension, and just assign to the original variable.